packages/luna/src/components/TutorCard/TutorCardV1.tsx

```diff
@@ -0,0 +1,216 @@
+                </Typography>
+              </div>
+            ) : null}
+
+            {rating ? (
+              <div className="tw-flex tw-flex-col">
+                <Typography
+                  element="caption"
+                  weight="regular"
+                  className="tw-text-natural-800"
+                >
+                  {intl("tutors.card.label.rating")}
+                </Typography>
+                <div className="tw-flex tw-flex-row tw-items-center tw-gap-[5px] tw-h-full">
+                  <Typography
+                    weight="semibold"
+                    className="tw-inline-block tw-text-natural-950 tw-text-xs tw-leading-3-half"
+                  >
+                    {rating}
+                  </Typography>
+                  <SStar
+                    width={15}
+                    height={15}
+                    className="[&>*]:tw-fill-warning-500"
+                  />
+                </div>
+              </div>
+            ) : null}
+          </div>
+        ) : null}
+        {topics ? (
+          <div className="tw-flex tw-mt-[15px] tw-gap-2 tw-flex-wrap tw-justify-start">
+            {topics.map((topic, idx, arr) => {
+              if (!isTutorStats && idx <= 7)
+                return (
+                  <Typography
+                    key={idx}
+                    element="tiny-text"
+                    weight="regular"
+                    className="tw-inline-block tw-text-natural-50 tw-bg-brand-700 tw-px-3 tw-py-2 tw-rounded-3xl"
+                  >
+                    {idx < 7 ? topic : null}
+                    {idx === 7
+                      ? intl("labels.plus", { value: arr.length - idx })
+                      : null}
+                  </Typography>
+                );
+
+              if (idx <= 4)
+                return (
+                  <Typography
+                    key={idx}
+                    element="tiny-text"
+                    weight="regular"
+                    className="tw-inline-block tw-text-natural-50 tw-bg-brand-700 tw-px-3 tw-py-2 tw-rounded-3xl"
+                  >
+                    {idx < 4 ? topic : null}
+                    {idx === 4
+                      ? intl("labels.plus", { value: arr.length - idx })
+                      : null}
+                  </Typography>
+                );
+            })}
+          </div>
+        ) : null}
+        <div className="tw-flex tw-flex-row tw-gap-3 tw-mt-4">
+          <Button
+            onClick={onBook}
+            className="tw-w-full"
+            type={ButtonType.Main}
+            variant={ButtonVariant.Primary}
+            size={ButtonSize.Tiny}
+          >
+            <Typography
+              element="caption"
+              weight="semibold"
+              className="tw-text-natural-50"
+            >
+              {intl("tutors.card.book-button.label")}
+            </Typography>
+          </Button>
+          <Button
```

Comment:
I think it will be better to convert this `button` to link and instead of using the `onOpenProfile` handler, we can use the `profileUrl`.


```suggestion
          <Link to={ProfileUrl}
            <Button
```

Reply:
Don't forget to remove the the `onOpenProfile` from the props. Also feel free to make this change to the other tutor card as well.